### PR TITLE
Simulation used for realtime docs

### PR DESCRIPTION
**Pull request by aidylewis**

### CODE REVIEW

File: src/test/scala/computerdatabase/ComputerWorld.scala [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import io.gatling.http.Predef._
+
+class ComputerWorld extends Simulation {
+
+  val httpProtocol = http
+    .baseURL("http://computer-database.gatling.io")
+    .acceptHeader("""text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8""")
+    .acceptEncodingHeader("""gzip, deflate""")
+    .acceptLanguageHeader("""en-gb,en;q=0.5""")
+    .contentTypeHeader("application/x-www-form-urlencoded")
```
Comment (slandelle) [inline src/test/scala/computerdatabase/ComputerWorld.scala]: remove this, it doesn't have to be sent for the get requests, and it's automatically generated for the post one

**Comment by aidylewis** [comment]
Shall I squash the commits into one?

**Comment by slandelle** [comment]
I can do it. I'll cherry-pick anyway.

**Comment by slandelle** [comment]
cherry-picked as e71a59c9bd9228265712583d6b642f4ead352f91

Thanks!

**Pull request closed without merging**